fitwsarah_frontend/src/views/ProfilePage/Account.js

```diff
@@ -58,7 +58,7 @@ function Profile() {
         if (accessToken) {
             getAppointmentsByUserId(extractAfterPipe(user.sub), status);
         }
-    }, [user, status]);
+    }, [status, accessToken]);
```

Comment:
Is user important here?

Reply:
No because if the accessToken is set the user will be set as well